vendor/vendor.json

```diff
@@ -1302,16 +1302,20 @@
 			"versionExact": "v1.18.0"
```

Comment:
Can you rebase this branch against `master`? You have already submitted a change that brings the vendor dependencies in line with the changes here, so there should be reverted after the rebase.